src/components/ClientMainParts/MainPart.vue

```diff
@@ -36,35 +36,23 @@
       </div>
     </div>
   </div>
-  <div v-else><CircleSpinner /></div>
 </template>
 
 <script>
-  import Button from '../Button';
-  import 'font-awesome/css/font-awesome.min.css';
-  import { mapState, mapActions } from 'vuex';
-
-  export default {
-    name: 'MainPart',
-    props: {
-      msg: String
-    },
-    components: {
-      Button
-    },
-    computed: mapState({
-      seminar: state => state.seminars.seminar,
-      loading: state => state.seminars.loading,
-    }),
-    methods: {
-      ...mapActions('seminars', [
-        'fetchCurrentSeminar',
-      ]),
-    },
-    created() {
-      this.fetchCurrentSeminar();
-    }
-  }
+import 'font-awesome/css/font-awesome.min.css';
```

Comment:
all css includes should be at root entry point file

Reply:
Done.